.gitignore

```diff
@@ -20,8 +20,8 @@
 *.byebug_history
 
 # Ignore application configuration
-/config/env/test.env
-/config/env/development.env
+/config/env/test.example.env
```

Comment:
You don't need to have `/config/env/test.example.env` and `/config/env/development.example.env` in .gitignore because you're putting your actual keys into `/config/env/test.env` and `/config/env/development.env`. Recall this [part](https://github.com/julianguyen/ifme#mandatory) of the README :P